rpc/wrtc_client.go

```diff
@@ -173,8 +173,9 @@ func dialWebRTC(
 		}
 		averageCallUpdateDuration := totalCallUpdateDuration / time.Duration(callUpdates)
 		// TODO: Potentially report these stats to sentry/some central location at some point.
-		logger.Debugw("ICE connected", "time_since_dial_start", time.Since(dialStart), "num_call_updates",
-			callUpdates, "average_duration", averageCallUpdateDuration, "max_call_update_duration", maxCallUpdateDuration)
+		logger.Debugw("ICE connected", "time_since_dial_start_ms", time.Since(dialStart).Milliseconds(), "num_call_updates",
+			callUpdates, "average_duration_ms", averageCallUpdateDuration.Milliseconds(), "max_call_update_duration_ms",
+			maxCallUpdateDuration.Milliseconds())
```

Comment:
These times were getting logged as nanoseconds; switched to milliseconds and added `ms` suffixes to fields to clarify.